### ISSUE TITLE
I got exit returncode 53 and exit returncode 1.

### ISSUE THREAD

**Issue by Pchaoz** [lutris/lutris#2641]
I saw another issue like this, but i doesn't solved me nothing, i can't install any game at lutris. Here's what i get when i try to install something. (Game: Legends of runeterra) (Distro: Debian 9)

> 2020-03-04 15:59:18,595: Running Mesa driver 13.0.6 on Mesa DRI Intel(R) Haswell Desktop  (0x412)
2020-03-04 15:59:18,595: GPU: 8086:0412 103C:18E7 using i915 drivers
2020-03-04 15:59:18,595: i386 libvulkan.so.1 missing (needed by vulkan)
2020-03-04 15:59:18,596: Vulkan is not available or your system isn't Vulkan capable
2020-03-04 15:59:18,597: Updating DXVK versions
2020-03-04 15:59:19,263: Updating D9VK versions
2020-03-04 15:59:25,784: Getting runner information for wine
2020-03-04 15:59:25,949: Downloading file 1 of 1
2020-03-04 15:59:32,473: Creating a win64 prefix in /home/super/LutrisGames/legends-of-runeterra
2020-03-04 15:59:37,752: Wine prefix creation is taking longer than expected...
2020-03-04 15:59:45,011: No user.reg found after prefix creation. Prefix might not be valid
2020-03-04 15:59:45,018: Creating a win64 prefix in /home/super/LutrisGames/legends-of-runeterra
2020-03-04 15:59:50,297: Wine prefix creation is taking longer than expected...
2020-03-04 15:59:57,557: No user.reg found after prefix creation. Prefix might not be valid
2020-03-04 15:59:57,595: Creating a win64 prefix in /home/super/LutrisGames/legends-of-runeterra
2020-03-04 16:00:02,876: Wine prefix creation is taking longer than expected...
2020-03-04 16:00:10,136: No user.reg found after prefix creation. Prefix might not be valid
2020-03-04 16:00:10,188: Creating a win64 prefix in /home/super/LutrisGames/legends-of-runeterra
2020-03-04 16:00:15,471: Wine prefix creation is taking longer than expected...
2020-03-04 16:00:22,731: No user.reg found after prefix creation. Prefix might not be valid
Running /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/wine /home/super/.cache/lutris/installer/legends-of-runeterra/lor_installer/live.exe
000b:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib64/wine/msvcrt.dll.so": /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib64/wine/msvcrt.dll.so)
000b:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\wineboot.exe") failed (error c000007a).
000b:err:module:LdrInitializeThunk Importing dlls for L"C:\\windows\\system32\\wineboot.exe" failed, status c0000135
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\CRYPT32.dll") not found
0009:err:module:import_dll Library CRYPT32.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\rpcrt4.dll") failed (error c000007a).
0009:err:module:import_dll Library rpcrt4.dll (which is needed by L"C:\\windows\\system32\\netapi32.dll") not found
0009:err:module:import_dll Library netapi32.dll (which is needed by L"C:\\windows\\system32\\DNSAPI.dll") not found
0009:err:module:import_dll Library DNSAPI.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\shlwapi.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\shlwapi.dll") failed (error c000007a).
0009:err:module:import_dll Library shlwapi.dll (which is needed by L"C:\\windows\\system32\\gdiplus.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\rpcrt4.dll") failed (error c000007a).
0009:err:module:import_dll Library rpcrt4.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") failed (error c000007a).
0009:err:module:import_dll Library ole32.dll (which is needed by L"C:\\windows\\system32\\gdiplus.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\gdiplus.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/ucrtbase.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/ucrtbase.dll.so)
0009:err:module:import_dll Loading library ucrtbase.dll (which is needed by L"C:\\windows\\system32\\gdiplus.dll") failed (error c000007a).
0009:err:module:import_dll Library gdiplus.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\rpcrt4.dll") failed (error c000007a).
0009:err:module:import_dll Library rpcrt4.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\ole32.dll") failed (error c000007a).
0009:err:module:import_dll Library ole32.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\RPCRT4.dll") failed (error c000007a).
0009:err:module:import_dll Library RPCRT4.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\shlwapi.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\shlwapi.dll") failed (error c000007a).
0009:err:module:import_dll Library shlwapi.dll (which is needed by L"C:\\windows\\system32\\SHELL32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\SHELL32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\comctl32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\usp10.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\usp10.dll") failed (error c000007a).
0009:err:module:import_dll Library usp10.dll (which is needed by L"C:\\windows\\system32\\comctl32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\imm32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\imm32.dll") failed (error c000007a).
0009:err:module:import_dll Library imm32.dll (which is needed by L"C:\\windows\\system32\\comctl32.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\comctl32.dll") failed (error c000007a).
0009:err:module:import_dll Library comctl32.dll (which is needed by L"C:\\windows\\system32\\aclui.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\user32.dll") not found
0009:err:module:import_dll Library user32.dll (which is needed by L"C:\\windows\\system32\\aclui.dll") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\aclui.dll") failed (error c000007a).
0009:err:module:import_dll Library aclui.dll (which is needed by L"C:\\windows\\system32\\SHELL32.dll") not found
0009:err:module:import_dll Library SHELL32.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:load_so_dll failed to load .so lib "/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so": /lib32/libm.so.6: version `GLIBC_2.27' not found (required by /home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/../lib/wine/msvcrt.dll.so)
0009:err:module:import_dll Loading library msvcrt.dll (which is needed by L"C:\\windows\\system32\\version.dll") failed (error c000007a).
0009:err:module:import_dll Library version.dll (which is needed by L"C:\\windows\\system32\\USER32.dll") not found
0009:err:module:import_dll Library USER32.dll (which is needed by L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe") not found
0009:err:module:LdrInitializeThunk Importing dlls for L"Z:\\home\\super\\.cache\\lutris\\installer\\legends-of-runeterra\\lor_installer\\live.exe" failed, status c0000135
Waiting on children
All children gone
Exit with returncode 53
2020-03-04 16:00:23,752: Creating a win64 prefix in /home/super/LutrisGames/legends-of-runeterra
2020-03-04 16:00:29,031: Wine prefix creation is taking longer than expected...
2020-03-04 16:00:36,291: No user.reg found after prefix creation. Prefix might not be valid
2020-03-04 16:00:36,302: The game doesn't have an executable
Running /home/super/.local/share/lutris/runtime/winetricks/winetricks --unattended d3dcompiler_43 d3dx9 wininet dotnet48
grep: /home/super/LutrisGames/legends-of-runeterra/*.reg: No such file or directory
------------------------------------------------------
WINEPREFIX INFO:
Drive C: total 12
drwxr-xr-x 3 super super 4096 Mar  4 15:58 .
drwxr-xr-x 4 super super 4096 Mar  4 15:58 ..
drwxr-xr-x 4 super super 4096 Mar  4 15:58 windows

Registry info:
/home/super/LutrisGames/legends-of-runeterra/*.reg:
------------------------------------------------------
------------------------------------------------------
/home/super/.local/share/lutris/runners/wine/lutris-5.2-x86_64/bin/wine cmd.exe /c echo '%ProgramFiles%' returned empty string, error message "" 
------------------------------------------------------
Waiting on children
All children gone
Exit with returncode 1
2020-03-04 16:00:39,666: No executable found at specified location /home/super/LutrisGames/legends-of-runeterra/drive_c/Riot Games/Riot Client/RiotClientServices.exe

**Comment by strycore** [comment]
Debian 9 is too old and won't run our Wine builds. Please update to Debian 10.
